attributes/network/9_account_features/cddl/1_account_multisig.cddl

```diff
@@ -19,10 +19,12 @@ account-multisig-transaction-submit = {
     0 => [9, 1, 0],
     1 => non-anonymous-identity,        ; Submitter
     2 => non-anonymous-identity,        ; Account source
-    3 => text,                          ; Description of the transaction
+    ? 3 => bstr,                        ; Description of the transaction
     4 => account-multisig-transaction,  ; The transaction submitted
-    5 => uint,                          ; The number of approvals needed.
-    6 => time,                          ; Expiration date time.
+    5 => account-multisig-token,        ; Token created by the submission
```

Comment:
How can one submit a multisig-transaction without having the `account-multisig-token`?

Reply:
Urgh, yeah. This should be in the transaction history but not if you create a new transaction. I'll make it optional and specify.